### ISSUE TITLE
TYPO3 v11?

### ISSUE THREAD

**Issue by s12r** [p2media/typo3-ext--be-acl#6]
Hello,
are there any plans to make typo3-ext--be-acl compatible to TYPO3 v11?

**Comment by phth** [comment]
I created a fork wich is compatible with typo3 11 in a functional sense (but not in terms of clean code): 
https://github.com/phth/typo3-ext--be-acl/commits/master
Could be a starting point :)

**Comment by gripsITGmbH** [comment]
Great @phth - we added some fixes to your code and created a Pullrequest to your Fork. Looking good so far.

Our fork can be found here: 
https://github.com/gripsITGmbH/typo3-ext--be-acl

**Comment by jokumer** [comment]
Even https://github.com/OpenSourceInternational/be_acl/tree/typo3-11-compatibility
is available. I'll eval both and return here

**Comment by jokumer** [comment]
IMO https://github.com/gripsITGmbH/typo3-ext--be-acl is better made

**Comment by jsleocadio** [comment]
Is it possible to update `composer.json` so can download the extension via composer?

**Comment by jokumer** [comment]
What is wrong with `composer.json`?
You can configure your composer to use this repo
`composer config repositories.be-acl vcs git@github.com:p2media/typo3-ext--be-acl.git`
and load with
`composer require p2media/be-acl:dev-master`

**Comment by jsleocadio** [comment]
I'm using Typo3 11.5 and when i use composer occur this problem:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires p2media/be-acl dev-master -> satisfiable by p2media/be-acl[dev-master].
    - p2media/be-acl dev-master requires typo3/cms-core ^10.4 -> found typo3/cms-core[v10.4.0, ..., v10.4.36] but it conflicts with your root composer.json require (^11.5.0).

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by jokumer** [comment]
You're right. Sorry. Thats why I prefer another fork / version from gripsITGmbH
`composer config repositories.be-acl vcs git@github.com:gripsITGmbH/typo3-ext--be-acl.git`

**Comment by jsleocadio** [comment]
Thank you, @jokumer ! Work just fine right now! In my project has too many groups and Typo3 yet don't work with various usergroups rights to access a page. This extension is awesome and will help me a lot.
